CalPatRec/src/TZClusterFinder_module.cc

```diff
@@ -790,24 +800,24 @@ namespace mu2e {
     const mu2e::ComboHit* test;
 
     // hit position variables
-    double hitX = 0.0;
-    double hitY = 0.0;
-    double testX = 0.0;
-    double testY = 0.0;
+    double hitX       = 0.0;
```

Comment:
not mandatory for this PR, but for the next time, you could write `double var(0.), var1(0.), var2(0.);`